helmfile.d/10-services.yaml

```diff
@@ -139,15 +139,37 @@ releases:
       - name: ingress.tls[0].hosts
         values: ["{{ .Values.server_name }}"]
 
+  - name: radar-kafka
+    chart: radar/radar-kafka
+    version: {{ .Values.radar_kafka_stack._chart_version }}
+    installed: {{ .Values.radar_kafka_stack._install }}
+    timeout: {{ add .Values.base_timeout .Values.radar_kafka_stack._extra_timeout }}
+    <<: *logFailedRelease
+    values:
+      - {{ .Values.radar_kafka_stack | toYaml | indent 8 | trim }}
+    set:
+      - name: kafka.replicas
```

Comment:
There is also `kafka.insyncReplicas` parameter for radar-kafka. Should it be also set here, as it might be dependent on `kafka.replicas`?